src/actions/ActionTypes.js

```diff
@@ -10,6 +10,11 @@ export default keyMirror({
   LOGIN_FAIL: '',
   LOGIN_LOGOUT: '',
 
+  PASSWORD_CHANGE_SUCCESS: '',
+  PASSWORD_CHANGE_FAIL: '',
+  PASSWORD_NEW_NEEDED_SUCCESS: '',
+  PASSWORD_NEW_NEEDED_FAIL: '',
```

Comment:
een actionType is pas nodig wanneer je er in een reducer ook effectief iets met doet

deze dingen veranderen de state niet... delete....